src/stdune/table.mli

```diff
@@ -0,0 +1,11 @@
+module type Key = sig type t val equal : t -> t -> bool val hash : t -> int end
```

Comment:
Could you please an odoc comment summarizing the purpose of this module?

Reply:
Added.